index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <script src="/js/tools.js"></script>
 
-    <script src="//unpkg.com/three"></script>
+    <!--<script src="//unpkg.com/three"></script>-->
```

Comment:
why commented and not removed?

Reply:
This is used for the clouds layer in the globe animation.  A lot of options were discussed for replacing what there is now but nothing permanent has been put in place so I'm keeping all optional stuff there.